packages/block-library/src/group/edit.native.js

```diff
@@ -13,6 +13,7 @@ import {
 	InnerBlocks,
 	withColors,
 } from '@wordpress/block-editor';
+
```

Comment:
You probably want to revert this change as well

Reply:
Youre rigth

Reply:
done